server/app/services/program/BlockDefinition.java

```diff
@@ -117,6 +117,15 @@ public boolean isFileUpload() {
         .anyMatch(questionType -> questionType.equals(QuestionType.FILEUPLOAD));
   }
 
+  @JsonIgnore
+  @Memoized
+  public boolean isAddress() {
```

Comment:
suggestion (non-blocking): 
```suggestion
  public boolean hasAddress() {
```

Reply:
enumerator blocks can only have a single enumerator question, whereas address questions can share the block with other questions